src/net/cert/internal/system_trust_store.cc

```diff
@@ -325,23 +325,27 @@ std::unique_ptr<SystemTrustStore> CreateSslSystemTrustStore() {
 
 namespace {
 
-// Copied from https://golang.org/src/crypto/x509/root_linux.go
+// Copied from https://go.dev/src/crypto/x509/root_linux.go
 // Possible certificate files; stop after finding one.
-constexpr std::array<const char*, 6> kStaticRootCertFiles = {
-    "/etc/ssl/certs/ca-certificates.crt",  // Debian/Ubuntu/Gentoo etc.
-    "/etc/pki/tls/certs/ca-bundle.crt",    // Fedora/RHEL 6
-    "/etc/ssl/ca-bundle.pem",              // OpenSUSE
-    "/etc/pki/tls/cacert.pem",             // OpenELEC
+constexpr const char* kStaticRootCertFiles[] = {
+    "/etc/ssl/certs/ca-certificates.crt",                 // Debian/Ubuntu/Gentoo etc.
+    "/etc/pki/tls/certs/ca-bundle.crt",                   // Fedora/RHEL
+    "/etc/ssl/ca-bundle.pem",                             // OpenSUSE
+    "/etc/pki/tls/cacert.pem",                            // OpenELEC
     "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem",  // CentOS/RHEL 7
     "/etc/ssl/cert.pem",                                  // Alpine Linux
 };
 
 // Possible directories with certificate files; stop after successfully
 // reading at least one file from a directory.
-constexpr std::array<const char*, 3> kStaticRootCertDirs = {
-    "/etc/ssl/certs",      // SLES10/SLES11, https://golang.org/issue/12139
-    "/etc/pki/tls/certs",  // Fedora/RHEL
-    "/system/etc/security/cacerts",  // Android
+constexpr const char* kStaticRootCertDirs[] = {
+    "/etc/ssl/certs",                                  // SLES10/SLES11, https://golang.org/issue/12139
+    "/etc/pki/tls/certs",                              // Fedora/RHEL
+    "/etc/pki/ca-trust/extracted/pem/directory-hash",  // Fedora/RHEL 9.5/10
```

Comment:
Not seeing this in crypto/x509/root_linux.go

Reply:
It is based on latest code in RHEL 9.5 and RHEL 10 beta. You can download the beta iso and test it.

Reply:
> ca-certificates trust store moved

The /etc/pki/tls/certs trust store is converted to a different format better optimized for OpenSSL. As a consequence, if you use the files in /etc/pki/tls/certs directly, switch to the /etc/pki/ca-trust/extracted directory, where the same data is stored. For example, software that accesses the trust bundle at /etc/pki/tls/certs/ca-bundle.crt should switch to using /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem instead. 

https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/10-beta/html/considerations_in_adopting_rhel_10/security

Reply:
![image](https://github.com/user-attachments/assets/e3570255-4f85-4205-bf64-b2da89ebfc98)

Reply:
Not seeing how Golang intends to support this change.

Reply:
Copy is cheat.

I don't know why you make double standards for Golang's change. It is shameful. Lol.

Reply:
Nobody is bringing PRs to Golang for this change, because by any common sense, Fedora will not break backward compatibility for no reason. 

https://src.fedoraproject.org/rpms/ca-certificates/blob/rawhide/f/update-ca-trust.8.txt#_261 /etc/pki/tls/certs::
	Contains symbolic links to the directory-hash format certificates generated by update-ca-trust command.